router-tests/testenv/testenv.go

```diff
@@ -693,6 +693,12 @@ func configureRouter(listenerAddr string, testConfig *Config, routerConfig *node
 				},
 			},
 			ForwardInitialPayload: true,
+			Authentication: config.WebSocketAuthenticationConfiguration{
```

Comment:
yes, I think it will probably break other tests, I have to fix it but I'm having a hard time running the dev-setup since I'm working on Windows, and the make commands seems to be compatible only with linux or IOS. Do you have any suggestion how can I make it work on Windows?

Reply:
Managed to run them in a virtual machine.